### PR TITLE
Feature/user to synagogue

### PR DESCRIPTION
**Pull request by ofirschliss**

### CODE REVIEW

File: django/webapp/models.py [hunk 1]
```diff
@@ -278,3 +277,34 @@ def get_aliya_precedence(self, on_date: HebrewDate) -> Optional[int]:
...
+
+    def _get_field_as_json(self, field: str) -> Optional[Dict]:
+        value = getattr(self, field)
+        if value is None:
+            return None
+        return {'id': value.pk, 'name': value.full_name}
+
+    @property
+    def father_json(self) -> Optional[Dict]:
+        return self._get_field_as_json('father')
```
Comment (brianmaissy) [inline django/webapp/models.py:293]: i would pass self.father as the parameter and avoid the `getattr`

File: django/webapp/models.py [hunk 2]
```diff
@@ -37,7 +37,6 @@ class CannotGetAliya(Exception):
 
 class Synagogue(models.Model):
     name = models.TextField()
-    admins = models.ForeignKey(Group, on_delete=models.CASCADE)
     member_creator = models.ForeignKey(User, on_delete=models.CASCADE)
```
Comment (brianmaissy) [inline django/webapp/models.py:40]: do we still need this mechanism?

Comment (ofirschliss) [reply django/webapp/models.py:40]: why not?
it's a very advanced feature and it was silly to write it that early, but we will need it if we will want to let someone add member without being able to do nothing else

Comment (brianmaissy) [reply django/webapp/models.py:40]: just because its sort of half implemented, but we can leave it if you want

**Pull request merged**